### ISSUE TITLE
[NPC] Tunnel Rat Kobolds have wrong speech dialogue

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5358]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Tunnel rat kobold's speech dialogue is that of the stonesplinter troggs
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Engage the kobolds in combat until one of them speaks
2. 
3. 

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/32976781/37772461-aa26dc2a-2e15-11e8-9a62-e48be5e8ef4a.png)
Please refer to the general chat log in the screenshot posted above

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
@wrathofthebug what should the  Tunnel Rat Kobold https://wowgaming.altervista.org/aowow/?npc=1202 say instead?

**Comment by Rushor** [comment]
nah forget it, i'm dump :D the text should be removed  - only those creatures should use that line sometimes https://wow.gamepedia.com/Stonesplinter_tribe

**Comment by wrathofthebug** [comment]
I think possible speech lines could be similar to those of other kobolds for example those near fargodeep mine in Elwynn Forest? Not too sure either haha couldn't find a source for their list of speech dialogues.

**Comment by Rushor** [comment]
yeah i could not find a list too :(
